docroot/modules/custom/mass_microsites/mass_microsites.module

```diff
@@ -38,17 +38,23 @@ function mass_microsites_node_update(NodeInterface $node) {
 function mass_microsites_preprocess_page(&$variables) {
   $params = \Drupal::routeMatch()->getParameters();
 
-    if ($params->has('node')) {
-      $node = $params->get('node');
-      /** @var MicrositeInterface */
-      $microsite = \Drupal::service('mass_microsites.nearest_microsite_lookup')->getNearestMicrosite($node);
-      $variables['microsite_info'] = [
-        'title' => $microsite->label(),
-        'url' => $microsite->getHome()->toUrl(),
-        'altTitle' => $microsite->label() . ' home page',
-        'hideImage' => FALSE,
-      ];
-    }
+  $node_full_view_routes = [
+    'entity.node.canonical',
+    'entity.node.revision',
+    'entity.node.preview',
+  ];
+
+  if (in_array(\Drupal::routeMatch()->getRouteName(), $node_full_view_routes)) {
+    $node = $params->get('node');
```

Comment:
Note that I think you'll need to pull the `node_preview` param for the `entity.node.preview` route (have I been bit by this before ... maybe).